satellite-software/include/BME680Sensor.h

```diff
@@ -8,7 +8,7 @@ class PersistedConfiguration;
 //
 // BME680 Sensor
 //
-#define BME680_I2C_ADDRESS            0x76
+#define BME680_I2C_ADDRESS            0x77
```

Comment:
The address of the BME680 as configured by the AmbaSat sensor as provided by AmbaSat is 0x76. Are you using the AdaFruit BME680 sensor? Regardless, the code should default to the AmbaSat configuration.

Reply:
Indeed, changed. 
This is based on my prototype hardware and I forgot to change this back.